main.cpp

```diff
@@ -0,0 +1,68 @@
+#include <iostream>
+#include "battery_health.h"
+#define CATCH_CONFIG_MAIN  // This tells Catch to provide a main() - only do this in one cpp file
+#include "test/catch.hpp"
+
+PassiveCooling passiveCooling;
+HighActiveCooling highActiveCooling;
+MediumActiveCooling mediumActiveCooling;
+SendToEmail sendToEmail;
+SendToController sendToController;
+BatteryHealth batteryHealth;
+bool isAlertMessageSend;
+
+TEST_CASE("PassiveCooling breach, send via email") {
+isAlertMessageSend = batteryHealth.checkAndAlert(&passiveCooling, &sendToEmail, 30);

```

Comment:
Test cases need to be added to check the message type of breach and the if the message is received by the intended alert target.